frontend/src/pages/home/home.tsx

```diff
@@ -125,10 +125,12 @@ export default class HomePage extends React.Component<HomePageProps, HomePageSta
                             <div className="notice-content">
                                 <p>These are all the messages we managed to collect! Happy Valentine's Day!</p>
                                 <p>これがすべての取集したメッセージですが！ハッピーバレンタイン！</p>
+                                <p style={{fontSize: 10}}>If you find any problems with the website, or if you would like to report a message, please contact Fishsauce#1324 on discord</p>
```

Comment:
If you want me to share the burden, you can put the google group email here instead/in addition.

Reply:
that would work too. if you wanna do that be my guest :D